List of Persian localized version of Iran geocode

afarsi opened the pull request:
I updated the list based on latest changes of /en/98.txt
I Removed the Increased lines related to North Khorasan and South Khorasan based on latest itu document in english version and applied changes to here also.

mizuni commented:
Hi,

We noticed quite a few of these actually have the same text though. 
What's the intention besides the changes you pointed out here? Are you just using different unicode?

Cheers,

-Mike

afarsi commented:
Hi Mike.
I'm sorry. Because the problem of MS Note pad, this things happened. I edited this list by MS Notepad.
And as result, notepad changed the Persian Yeh char with Arabic yeh char! This problem is one of problems in non standard applications which aren't compatible with Persian standard Unicode chars.
As result, github thinked i changed All of the lines!!! Because many lines contaning Persian.yeh char.
I edited the 98.txt with github online editor and fixed that problem.

Now if you look at Files changed section, you can understand what things are changed.
Changes which I applied to file are:

I only removed duplicate lines related to North Khorasan province. This changes also applied to English file by me. If you look at my pool request for Iran geocode, you will understand I removed only "North Khorasan" duplicate lines. "North Khorasan" is one of Iran's provinces(states) and have only one code, two lines and codes in 98.txt file are duplicate.
These changes are based on latest itu document(For more info look at my pool request Iran geocode)

behdad commented:
I fixed this and combined with the English change into https://github.com/googlei18n/libphonenumber/pull/1185
